src/utils/__tests__/index.ts

```diff
@@ -219,20 +221,34 @@ describe('stringToAccountKey and accountKeyToString', () => {
   });
 
   test('stringToAccountKey should convert a string to a polkadot AccountKey object', () => {
-    const value = 'someAccountKey';
+    const value = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
     const fakeResult = ('convertedAccountKey' as unknown) as AccountKey;
     const context = polkadotMockUtils.getContextInstance();
+    const decodeAddressResult = utilsCrypto.decodeAddress(value);
 
-    mockCreateType.withArgs(context.polymeshApi.registry, 'AccountKey', value).returns(fakeResult);
+    sinon
+      .stub(utilsCrypto, 'decodeAddress')
+      .withArgs(value)
+      .returns(decodeAddressResult);
+
+    mockCreateType
+      .withArgs(context.polymeshApi.registry, 'AccountKey', decodeAddressResult)
+      .returns(fakeResult);
 
     const result = stringToAccountKey(value, context);
 
     expect(result).toBe(fakeResult);
   });
 
   test('accountKeyToString should convert a polkadot AccountKey object to a string', () => {
-    const fakeResult = 'someAccountKey';
+    const fakeResult = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
     const accountKey = polkadotMockUtils.createMockAccountKey(fakeResult);
+    const encodeAddressResult = utilsCrypto.encodeAddress(u8aToString(accountKey));
```

Comment:
```suggestion
    const fakeResult = 'someAccountId';
    const accountKey = polkadotMockUtils.createMockAccountKey(fakeResult);
    const encodedValue = ('encodedAccountId' as unknown) as Uint8Array;
```

Reply:
The cast doesn't work as expected:  `Non-base58 character`